### ISSUE TITLE
Should I define sql functions myself?

### ISSUE THREAD

**Issue by liumiao-p** [Audi-1/sqli-labs#28]
sql-connection目录下找不到定义操作sql的一系列函数的php文件，比如mysql_connect(),mysql_query()等等，是不是需要读者自行定义？

**Comment by liumiao-p** [comment]
sql-connection目录下找不到定义操作sql的一系列函数的php文件，比如mysql_connect(),mysql_query()等等，是不是需要读者自行定义？

**Comment by psi-cmd** [comment]
No, just follow this pull request https://github.com/Audi-1/sqli-labs/pull/19#issue-220118610
Problem is that mysql_connect is banned from php-7.x, the fork of the pull request did the compatible work.

By the way, you may need to `apt install php7.x-mysql` and modify the `php.ini` in `/etc/php/7.3/apache2` to enable the mysql extention. Then restart related services.
